config/routes.rb

```diff
@@ -114,6 +114,7 @@
         get(:healthcheck)
         get(:enrollment_status)
         get(:rating_info)
+        get(:facilities)
```

Comment:
Good question. I always thought of it as the version of the endpoint would always start as v0 even if the upstream call it makes is from a v1 endpoint. Since this is the first Health Care Applications is creating a facilities endpoint, I'd still lean towards making this a v0. If there was already a health care application endpoint that integrated with the v0 LH facilities API then I'd definitely version the new one at v1.